### PR TITLE
Add ReadPlate

### PR DESCRIPTION
**Pull request by LauLauThom**
Done on request of the user who is not familiar with github
See on the forum
https://forum.image.sc/t/readplate-3-0-release/42008/5

### CODE REVIEW

File: sites.yml [hunk 1]
```diff
@@ -1570,9 +1570,9 @@ sites:
       PixFRET - an ImageJ plug-in for FRET calculation
       which can accommodate variations in spectral bleed-throughs
     maintainers:
-      - "[Jan Eglinger](https://imagej.net/User:Eglinger)"
-
-  - name: "PQCT"
+      - "[Jan Eglinger](https://imagej.net/User:Eglinger)
```
Comment (frauzufall) [inline sites.yml:1573]: Can you please re-add the missing quote there? Thanks :)

File: sites.yml [hunk 2]
```diff
@@ -1570,9 +1570,9 @@ sites:
...
+ 
+ - name: "PQCT"
```
Comment (frauzufall) [inline sites.yml:1575]: Can you re-add the space before the `-` to keep the indentation?

File: sites.yml [hunk 3]
```diff
@@ -1639,7 +1639,22 @@ sites:
...
+      multi-well plates of 6 (3x2), 12 (4x3), 24 (6x4), 48 (8x6) and 96 (12x8) 
+      wells. An new user-friendly interface facilitates image analysis. This 
+      version allows new user-defined features to optimize blank correction. 
+      Multiple readings of the same plate has also considerably been 
+      facilitated, appending the full parameter file to the results and log 
+      file.
+    maintainers:
+      - "José María Delfino (delfino@qb.ffyb.uba.ar)"
+      
+ - name: "ResultsToExcel"
```
Comment (frauzufall) [inline sites.yml:1657]: I think there's a space missing before the `-` here too ^^

**Review by frauzufall — CHANGES_REQUESTED** [review]
Thanks for doing this!! I guess the test fails because of the deleted quote, and I was picky about the indentation, hope you don't mind :)

**Comment by LauLauThom** [comment]
Thanks for the quick feedback, lessons learned: don't use the online editor :P

**Comment by frauzufall** [comment]
:sweat_smile: Thanks, merged.

**Pull request merged**